content/800-data-platform/100-accelerate/200-getting-started.mdx

```diff
@@ -15,20 +15,21 @@ toc: true
 To get started with Accelerate, you will need the following:
 
 - A GitHub account.
-- A project that uses [Prisma Client](/concepts/components/prisma-client) `4.16.1` or higher. (If your project is using interactive transactions, you need to use `5.1.1` or higher.)
+- A project that uses [Prisma Client](/concepts/components/prisma-client) `4.16.1` or higher. If your project is using interactive transactions, you need to use `5.1.1` or higher.
+  > We **recommend** using the latest version of Prisma.
```

Comment:
if you want to include the recommendation, let's just add it in parantheses. otherwise it creates a lot of visual noise IMO,

```suggestion
- A project that uses [Prisma Client](/concepts/components/prisma-client) `4.16.1` or higher. If your project is using interactive transactions, you need to use `5.1.1` or higher. (We always recommend using the latest version of Prisma.)
```